var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -25,7 +25,7 @@
 from spack import *
 
 
-class Fftw(Package):
+class Fftw(AutotoolsPackage):
```

Comment:
This change should be undone. Otherwise, the package will run `./configure` and `make` before getting to the `install()` method.